toolkit/scripts/srpm_pack.mk

```diff
@@ -58,7 +58,7 @@ $(STATUS_FLAGS_DIR)/build_srpms.flag: $(local_specs) $(local_spec_dirs) $(SPECS_
 	touch $@
 else
 $(STATUS_FLAGS_DIR)/build_srpms.flag: $(local_specs) $(local_spec_dirs) $(local_sources) $(SPECS_DIR) $(go-srpmpacker)
-	GODEBUG=x509ignoreCN=0 $(go-srpmpacker) \
+	$(go-srpmpacker) \
```

Comment:
Just confirming - Are we safe to remove the `GODEBUG` compat flag now?https://github.com/microsoft/CBL-Mariner/pull/420 Is this safe because we have the new source blob store?

Reply:
It's been safe for a while, we just missed to clean this up. We've updated our source servers' certificates to use the SAN (Subject Alternative Name) field and not only the CN (Common Name) field and that was the fix, which made this workaround unnecessary anymore and we should make sure we don't ignore these errors in case they happen in the future.